### ISSUE TITLE
a11y: no focus trap when mobile menu is opened

### ISSUE THREAD

**Issue by wojtekmaj** [npmx-dev/npmx.dev#564]
Steps to reproduce:
- Small screen
- Tab tab tab... until you're at the Menu button
- Activate it
- Menu opens
- Tab again

Expected: First menu item focued

Actual: Some link underneath the menu gets focused
